src/extras/PMREMGenerator.js

```diff
@@ -438,7 +438,7 @@ function _createRenderTarget( params ) {
 function _setViewport( target, x, y, width, height ) {
 
 	var viewport = new Vector4( x, y, width, height );
-	viewport.addScalar( 0.5 ).divideScalar( _renderer.getPixelRatio() );
+	viewport.addScalar( 0.5 );
```

Comment:
>viewport.addScalar( 0.5 );

I do not think this is correct. You should not need it. (What is your reason?)

In which case, I'd just set the viewport directly using the passed args.

Reply:
I was not sure about this line. The addition of ` 0.5` was also done in the initial version.

Reply:
The offset was introduced on https://github.com/mrdoob/three.js/pull/18326 as a fix for the incorrect mipmap coordinates on devices with floating point pixel ratio. I'm not sure if these are necessary after the changes.

Reply:
Tested locally and it appears the offset is no longer necessary after the changes.

Reply:
And it shouldn't. You should be passing in integer values to the method.

Reply:
Okay, let's remove this line 👍